### PR TITLE
Feat/onboarding video step connection

### PR DESCRIPTION
**Pull request by sean-brydon**
## What does this PR do?
Adds a new step to onboarding to allow the user to install video apps in the installation process 

Fixes # (issue)

Loom: https://www.loom.com/share/2e1db44051ec479b80df708ad9cda637

### CODE REVIEW

File: apps/web/components/getting-started/components/AppConnectionItem.tsx
Comment (sean-brydon) [inline apps/web/components/getting-started/components/AppConnectionItem.tsx:1]: Made this more genetic - useable in more places across getting started than just calendar

File: packages/prisma/enums/index.ts
Comment (sean-brydon) [inline packages/prisma/enums/index.ts:1]: prettier

File: packages/ui/components/scrollable/ScrollableArea.tsx
Comment (sean-brydon) [inline packages/ui/components/scrollable/ScrollableArea.tsx:1]: WIP scrollable area that adds overflow highlight - couldnt get it to work but its not important to this PR. 

Our scrollbar custom class always shows scrollbar if there is overflow (Good enough indicator for now)

File: apps/web/components/getting-started/steps-views/ConnectedVideoStep.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+              <li className="flex w-full flex-row justify-center border-b-0 py-6" key={index}>
+                <SkeletonAvatar className="mx-6 h-8 w-8 px-4" />
+                <SkeletonText className="ml-1 mr-4 mt-3 h-5 w-full" />
+                <SkeletonButton className="mr-6 h-8 w-20 rounded-md p-5" />
+              </li>
+            );
+          })}
+        </ul>
+      )}
+      <button
```
Comment (CarinaWolli) [inline apps/web/components/getting-started/steps-views/ConnectedVideoStep.tsx:59]: Should we disable 'Next Step' when no video app was connected yet? As we do it for Calendars, if not we can remove the 'Set up later' as 'Next Step' would do the same

Screenshot 2023-05-11 at 14 08 56

File: apps/web/components/getting-started/steps-views/ConnectedVideoStep.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                    />
+                  )}
+                </li>
+              );
+            })}
+        </List>
+      )}
+
+      {isLoading && (
+        <ul className="bg-default divide-subtle border-subtle divide-y rounded-md border p-0 dark:bg-black">
```
Comment (CarinaWolli) [inline apps/web/components/getting-started/steps-views/ConnectedVideoStep.tsx:47]: Should we make this a reusable component? Looks like the calendar setup step has the same skeleton loaders

**Review by CarinaWolli — COMMENTED** [review]
Looks good 👍 I left some minor nonblocking comments

onboarding e2e tests are failing though

File: apps/web/components/getting-started/steps-views/ConnectedVideoStep.tsx (hunk 1 above)
Comment (sean-brydon) [reply apps/web/components/getting-started/steps-views/ConnectedVideoStep.tsx:59]: Good shout! On it!

I actually thought about this when testing just slipped my mind

**Review by CarinaWolli — APPROVED** [review]
Looks good 🙌🏻

**Comment by Jaibles** [comment]
great work, thanks @sean-brydon !

File: apps/web/pages/getting-started/[[...step]].tsx [hunk 3]
```diff
@@ -122,12 +134,15 @@ const OnboardingPage = (props: IOnboardingPageProps) => {
 
               {currentStep === "connected-calendar" && <ConnectedCalendars nextStep={() => goToIndex(2)} />}
 
+              {currentStep === "connected-video" && <ConnectedVideoStep nextStep={() => goToIndex(3)} />}
```
Comment (zomars) [inline apps/web/pages/getting-started/[[...step]].tsx:137]: Just a heads up, this is what broke the signup check in Checkly

**Pull request merged**